### DISCUSSION TITLE
Thrustmaster Throttle Mapping Issue

### DISCUSSION THREAD

**Discussion by dnoe10** [flybywiresim/aircraft#5221] [Help] [answered]
The Thrustmaster TCA Sidestick Airbus Edition includes an integrated throttle with [two different modes](https://ts.thrustmaster.com/download/accessories/manuals/TCA_Sidestick/TCA_Sidestick_Throttle_modes.pdf). In one mode, the very bottom of the throttle maps to 0%, while the top maps to 100%. In the other mode, the detent, which is located around 20% of the way from the bottom, maps to 0%, and sliding the throttle below the detent activates button 17, which can be bound within the game to activate the reverse thrust.

I've had this stick since the game came out and have used it in the second mode on a number of aircraft with no issues. At some point, the mappings for the A32NX changed such that idle is now at the 50% position and switching between the two modes on the stick has no effect. This is only happening for the A32NX. The throttle works exactly as before on the stock A320N and other aircraft such as the B787. I saw that there is a calibration feature in the FlyPad but there I was just able to calibrate the position of the in-game detents. What I need to do is change the position of idle. How can I do that? Or, is this a bug?

**Comment by tracernz** [comment, marked as answer]
Idle is one of the detents you get to calibrate. You will need to disable reverse on axis in the calibration screen, because reverse is not on your axis (it's on button 17 instead). Note that you will not be able to use reverse idle with this config (Airbus SOP on decelerating through 70 knots is to move thrust levers to reverse idle).

**Reply by dnoe10** [reply]
It was the "Reverser on Axis" setting that was tripping me up. Thanks!
